app/javascript/packs/contact-form-navigation-alert.js

```diff
@@ -0,0 +1,23 @@
+$(() => {
+   $("body").attr("data-turbolinks", false);
+
+   const formElement = $("form");
+   const originalState = formElement.serialize();
+   formElement.on("submit", onSubmit)
+
+   function shouldNavigateDirectly() {
+      const currentState = $("form").serialize();
+
+      $("*").blur();
```

Comment:
Just curious what this was for?

Reply:
If you decide to stay on the page, the clicked element remained focused. So, I clear the focus of all elements.